pkg/resources/metrics.go

```diff
@@ -149,6 +149,26 @@ func ReconcileRedisAlerts(ctx context.Context, client k8sclient.Client, inst *v1
 	return v1alpha1.PhaseCompleted, nil
 }
 
+// CreateSmtpSecretExists creates a PrometheusRule to alert if the rhmi-smtp-secret is present
+// the ocm sendgrid service creates a secret automatically this is a check for when that service fails
+func CreateSmtpSecretExists(ctx context.Context, client k8sclient.Client, cr *v1alpha1.RHMI) (v1alpha1.StatusPhase, error) {
+	alertName := "SendgridSmtpSecretExists"
+	ruleName := "sendgrid-smtp-secret-exists-rule"
+	alertExp := intstr.FromString(
+		fmt.Sprintf("absent(kube_secret_info{namespace='%s',secret='redhat-rhmi-smtp'} == 1)", cr.Namespace),
+	)
+	alertDescription := fmt.Sprintf("The Sendgrid SMTP secret has not been created in the %s namespace and may need to be created manualy", cr.Namespace)
+	labels := map[string]string{
+		"severity": "warning",
```

Comment:
Should this be critical :thinking: as it will indicate something has gone wrong with the sendgrid service, and the cluster will need creds manually added

Reply:
Maybe , at the moment CS-SRE provision clusters for customers,  so they would should see it without the need for a page. If we ever get to the case where the customer is provisioning for themselves it would need to be critical.  I think warning is good for the time being. WDYT @ciaranRoche

Reply:
Yeah that makes sense to me